pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -66,7 +69,7 @@ func httpsSupported(t *testing.T) bool {
 }
 
 func goTLSSupported() bool {
-	return runtime.GOARCH == "amd64"
+	return runtime.GOARCH == "amd64" && config.New().EnableRuntimeCompiler
```

Comment:
Indeed Go TLS supports CO-RE. I don't think it will matter much because of the current way the tests are structured (see https://github.com/DataDog/datadog-agent/blob/9d2ec90a908cae66000d092431b5575b38d48521/pkg/network/tracer/tracer_usm_linux_test.go#L594-L639), but it won't hurt to add CORE to the conditional so I'll update it